tiatoolbox/models/engine/engine_abc.py

```diff
@@ -3,13 +3,24 @@
 
 import numpy as np
 
+from tiatoolbox.wsicore.wsimeta import Units
 
-class IOConfigABC(ABC):
-    """Define an abstract class for holding predictor I/O information.
+
+class ModelIOConfigABC(ABC):  # noqa: B024
+    """Defines an abstract class for holding a CNN model's I/O information.
```

Comment:
Not necessarily a CNN?

Reply:
DL model?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/618/commits/285d99aa15c252868ae6a7193c7ea25a749730bc